student.rb

```diff
@@ -1,4 +1,7 @@
+require_relative 'high_fivable'
+
 class Student
+  include HighFivable
   attr_reader :age, :phase
   attr_accessor :name
```

Comment:
What does the Student class have in common with the Teacher classes? How can we refactor it?